test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ServerVersionHelperTests.cs

```diff
@@ -151,6 +160,19 @@ private void VerifyCalculateServerType(string serverVersion, SqlServerType expec
             Assert.Equal(expected, actual);
         }
 
+        private void VerifyCalculateServerTypeForEngineEdition(int engineEdition, SqlServerType expected)
+        {
+            ServerInfo serverInfo = new ServerInfo
+            {
+                EngineEditionId = engineEdition
+            };
+
+            SqlServerType actual = ServerVersionHelper.CalculateServerType(serverInfo);
+            string errorString = "Verify server type based on Engine Edition. Actual value: " + actual + ", Expected value: " + expected; 
```

Comment:
nit - but interpolated strings make this a lot easier to work with 

$"Verify server type based on Engine Edition. Actual value: {actual}, Expected value: {expected}"